### DISCUSSION TITLE
Scenario "Lhasa": Easy Math

### DISCUSSION THREAD

**Discussion by fduran** [SadServers/sadservers#23] [Scenarios]
Discussions about the scenario [Lhasa](https://github.com/fduran/sadservers/tree/main/scenarios/lhasa)

**Comment by toro-** [comment]
Unable to start this scenario.

The prompt never allows input, only shows cursor (no BASH prompt). Other scenarios I've tested so far appear to be OK.

Anyone else seen this?

*EDIT*: It appears to have come up after waiting/reloading the console window multiple times

**Comment by fduran** [comment]
@toro- hello, sometimes (seems rare) the shell window is not responsive and we are investigating and trying to reproduce. If you are so kind please let us know at @info@sadservers.com what browser/OS you are using, your SadServers user name and approximate time (UTC) of the occurrence.

**Reply by toro-** [reply]
Have followed up via email
